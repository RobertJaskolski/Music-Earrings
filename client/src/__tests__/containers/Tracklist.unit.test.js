describe('Tracklist container', () => {
  test('should return', () => {});
});
