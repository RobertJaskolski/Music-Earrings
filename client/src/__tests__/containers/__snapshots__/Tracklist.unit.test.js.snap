// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tracklist container should render component 1`] = `
<Tracklist
  MyAPIGetRecommendations={[Function]}
  SpotifyCreatePlaylist={[Function]}
  SpotifyGetRecommendations={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
/>
`;
